unity/src/test/scala/io/delta/unity/UCCatalogManagedClientSuite.scala

```diff
@@ -230,4 +230,19 @@ class UCCatalogManagedClientSuite extends AnyFunSuite with UCCatalogManagedTestU
     assert(builderTableProperties.get("foo") == "bar")
   }
 
+  test("buildCreateTableTransaction uses UC committer") {
+    // ===== GIVEN =====
+    val ucClient = new InMemoryUCClient("ucMetastoreId")
+    val ucCatalogManagedClient = new UCCatalogManagedClient(ucClient)
+
+    // ===== WHEN =====
+    val createTableTxnBuilder = ucCatalogManagedClient
+      .buildCreateTableTransaction("ucTableId", baseTestTablePath, testSchema, "test-engine")
+      .asInstanceOf[CreateTableTransactionBuilderImpl]
+
+    // ===== THEN =====
+    val committerOpt = createTableTxnBuilder.getCommitterOpt
+    assert(committerOpt.get().isInstanceOf[UCCatalogManagedCommitter])
```

Comment:
nit but I think this assert could have just been added to the previous test instead of a whole new one